app/src/main/java/com/example/smsrc/commands/utils/Commands.java

```diff
@@ -4,5 +4,5 @@
 import com.example.smsrc.commands.model.ModeSwitchCommand;
 
 public class Commands {
-    public static Command modeSwitchCommand = new ModeSwitchCommand();
+    //public static Command modeSwitchCommand = new ModeSwitchCommand();
```

Comment:
Why is this commented?

Reply:
this class never used.
i comment this line to compile since I have changed the constructor of ModeSwitchCommand

Reply:
If it is not used and has no side effect please remove it